model/trigger/version.go

```diff
@@ -169,3 +148,28 @@ func (t *versionTriggers) versionRegression(sub *event.Subscription) (*notificat
 	}
 	return nil, nil
 }
+
+func MakeVersionSelectors(v version.Version) []event.Selector {
```

Comment:
If you're going to do this, then please modify versionTriggers.Selectors to call this function, and make it a pointer.

Reply:
It calls this already. I think this is better as not a pointer because I want to force the caller to validate that the version exists